scripts/configure/zowe-configure.sh

```diff
@@ -99,25 +117,6 @@ chmod a+x "${ZOWE_ROOT_DIR}/scripts/zowe-support.sh"
 echo "Attempting to setup Zowe Proclib ... "
 . $CONFIG_DIR/zowe-configure-proclib.sh
 
-# TODO LATER - same as the above - zss won't start with those permissions,
-sed -e "s#{{root_dir}}#${ZOWE_ROOT_DIR}#" \
-  -e "s#{{zosmf_admin_group}}#${ZOWE_ZOSMF_ADMIN_GROUP}#" \
```

Comment:
Why has this code block been moved and bits been deleted?

Reply:
BAU changes happened in between.  Fixing this now.